source/agora/common/SCPHash.d

```diff
@@ -60,3 +61,11 @@ public uint512 getHashOf (uint64_t slot_idx, ref const(Value) prev, uint32_t has
 {
     return uint512(hashMulti(slot_idx, prev[], hash, round_num, value[]));
 }
+
+/// Ditto
+public uint512 getHashOf (uint64_t slot_idx, ref const(Value) prev, uint32_t hash,
+    int32_t round_num, ref const(NodeID) node_id, ref const(AgoraHash) random_seed)
+{
+    return uint512(hashMulti(slot_idx, prev[], hash, round_num, node_id,
```

Comment:
There is no need for this function. You can move this entire call inline into `computeHashNode`

Reply:
Yes I can, but It's overloaded from the `getHashOf` and it's not bad to show that it's overloaded for the new feature.

Reply:
There is a difference. Those other overloads are used by C++ code. This function is not.